web/rest/brand/features/provider/invoiceTemplate/putInvoiceTemplate.feature

```diff
@@ -34,3 +33,19 @@ Feature: Update invoice templates
           "global": false
       }
     """
+
+  Scenario: Can not update a global invoice template
+    Given I add Brand Authorization header
+     When I add "Content-Type" header equal to "application/json"
+      And I add "Accept" header equal to "application/json"
+      And I send a "PUT" request to "/invoice_templates/2" with body:
+    """
+      {
+          "name": "Updated",
+          "description": "updated global invoice template",
+          "template": "global body updated",
+          "templateHeader": "global header updated",
+          "templateFooter": "global footer updated"
+      }
+    """
+    Then the response status code should be 400
```

Comment:
Note this is a 400 instead of 403 response, because entity sanitize is throwing an exception that avoid this scenario, so API acl is not actually being used to avoid this update.

Reply:
API require entity instance before they can be run. You can add response code to the sanitize exception if you want.

`throw new \DomainException('bla bla bla ', 403);`